plugins/multipath/mp-plugin-ps.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Dummy Kernel plugin with whole set of policy sets (RMT, PFT, DTP, DTCP)
+ *
+ *    Leonardo Bergesio <leonardo.bergesio@i2cat.net>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ */
+
+#include <linux/export.h>
+#include <linux/module.h>
+#include <linux/string.h>
+
+#define RINA_PREFIX "pff-multipath"
+#define RINA_PFF_MULTIPATH_NAME "multipath"
+
+#include "logs.h"
+#include "rds/rmem.h"
+#include "rmt-ps.h"
```

Comment:
This include can be removed. It would be probably good to check if there are some others